src/app/features/benefits-bundle/benefits-training-discounts/benefits-training-discounts.component.spec.ts

```diff
@@ -0,0 +1,84 @@
+import { HttpClientTestingModule } from '@angular/common/http/testing';
+import { ActivatedRoute, RouterModule } from '@angular/router';
+import { RouterTestingModule } from '@angular/router/testing';
+import { BreadcrumbService } from '@core/services/breadcrumb.service';
+import { EstablishmentService } from '@core/services/establishment.service';
+import { PagesService } from '@core/services/pages.service';
+import { MockActivatedRoute } from '@core/test-utils/MockActivatedRoute';
+import { MockBreadcrumbService } from '@core/test-utils/MockBreadcrumbService';
+import { MockEstablishmentService } from '@core/test-utils/MockEstablishmentService';
+import { MockPagesService } from '@core/test-utils/MockPagesService';
+import { SharedModule } from '@shared/shared.module';
+import { render } from '@testing-library/angular';
+
+import { BenefitsTrainingDiscountsComponent } from './benefits-training-discounts.component';
+
+describe('BenefitsTrainingDiscountsComponent', () => {
+  const pages = MockPagesService.pagesFactory();
+
+  async function setup() {
+    const { fixture, getByText, queryByText } = await render(BenefitsTrainingDiscountsComponent, {
+      imports: [SharedModule, RouterModule, RouterTestingModule, HttpClientTestingModule],
+      providers: [
+        { provide: PagesService, useClass: MockPagesService },
+        { provide: EstablishmentService, useClass: MockEstablishmentService },
+        { provide: BreadcrumbService, useClass: MockBreadcrumbService },
+        {
+          provide: ActivatedRoute,
+          useValue: new MockActivatedRoute({
+            snapshot: {
+              data: {
+                pages,
+              },
+            },
+          }),
+        },
+      ],
+    });
+
+    const component = fixture.componentInstance;
+    return {
+      component,
+      fixture,
+      getByText,
+      queryByText,
+    };
+  }
+
+  it('should render a FirstLoginWizardComponent', async () => {
```

Comment:
```suggestion
  it('should render a BenefitsTrainingDiscountsComponent', async () => {
```
Do you want to rename this?